nixos/modules/virtualisation/xen-dom0.nix

```
@@ -1,27 +1,23 @@
 # Xen hypervisor (Dom0) support.
-
 { config, lib, pkgs, ... }:
-
-with lib;
-
 let
   cfg = config.virtualisation.xen;
 in
 
 {
   imports = [
-    (mkRemovedOptionModule [ "virtualisation" "xen" "qemu" ] "You don't need this option anymore, it will work without it.")
-    (mkRenamedOptionModule [ "virtualisation" "xen" "qemu-package" ] [ "virtualisation" "xen" "package-qemu" ])
+    (lib.mkRemovedOptionModule [ "virtualisation" "xen" "qemu" ] "You don't need this option lib.anymore, it will work without it.")
```

```suggestion
    (lib.mkRemovedOptionModule [ "virtualisation" "xen" "qemu" ] "You don't need this option anymore, it will work without it.")
```
Preeeeety sure that's not a library. :)

You might want to drop xen-dom0.nix from this PR entirely, as it'll all be overwritten by #324911. That's completely optional though; if it proves a hassle to modify the script you're using to produce this treewide, I'll deal with the merge conflict later.

---

then I will leave it out for now. thanks for the heads up.